frontends/concrete-python/examples/game_of_life/game_of_life.py

```diff
@@ -0,0 +1,500 @@
+import sys
+import time
+
+# Hide pygame prompt
+from os import environ
+
+import numpy as np
+
+environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
+
+# ruff: noqa:E402
+import argparse
+
+# ruff: noqa:E402
+import pygame
+
+# ruff: noqa:E402
+from concrete import fhe
+
+
+# Function to workaround the miss of padding in CP
+def by_hand_padding(original_grid, res):
+    padded_res = fhe.zeros(original_grid.shape)
+
+    original_grid_shape = original_grid.shape
+    assert padded_res.shape[0:2] == (1, 1)
+    padded_res[0, 0, 1 : original_grid_shape[2] - 1, 1 : original_grid_shape[3] - 1] = res
+
+    assert original_grid.shape == padded_res.shape
+
+    return padded_res
+
+
+# Function to workaround the miss of padding in CP
+def conv_with_hand_padding(grid, weight, do_padded_fix):
+    convoluted_grid = fhe.conv(
+        grid,
+        weight.reshape(1, 1, *weight.shape),
+        strides=(1, 1),
+        dilations=(1, 1),
+        group=1,
+        pads=(1, 1, 1, 1) if not do_padded_fix else (0, 0, 0, 0),
+    )
+
+    if do_padded_fix:
+        convoluted_grid = by_hand_padding(grid, convoluted_grid)
+
+    return convoluted_grid
+
+
+# Function for Game of Life
+@fhe.compiler({"grid": "encrypted"})
+def update_grid_method_3b(grid):
+    # Method which uses two first TLU of 3 bits and a third TLU of 2 bits
+
+    weights_method_3b_a = np.array(
+        [
+            [1, 1, 1],
+            [1, 0, 1],
+            [1, 1, 0],
+        ]
+    )
+    weights_method_3b_b = np.array(
+        [
+            [0, 0, 0],
+            [0, 0, 0],
+            [0, 0, 1],
+        ]
+    )
+    table_next_cell_3b_a = [i if i <= 3 else 4 for i in range(8)]
+    table_next_cell_3b_b = [i - 1 if i in [2, 3] else 0 for i in range(6)]
+    table_next_cell_3b_c = [int(i in [2, 3]) for i in range(4)]
+
+    table_cp_next_cell_3b_a = fhe.LookupTable(table_next_cell_3b_a)
+    table_cp_next_cell_3b_b = fhe.LookupTable(table_next_cell_3b_b)
+    table_cp_next_cell_3b_c = fhe.LookupTable(table_next_cell_3b_c)
```

Comment:
Why not putting the values directly in `fhe.LookupTable` call?

Reply:
(same as below)

Reply:
And maybe they can be constructed where they're used, as it'd be easier to associate values with the operation maybe?

Reply:
I am not a big fan of mixing code and constants. I think it's easier when you have just code, to know how big it is

Reply:
As you wish. To me, creating the values where they're used is easier to read.

Reply:
We have different opinions here.